ContextDesktop/src/test/java/com/git/migi_1/Context/entity/TestPlatform.java

```diff
@@ -68,7 +68,7 @@ public void setUp() throws Exception {
         Mockito.when(assetManager.loadModel(Mockito.anyString())).thenReturn(model);
         Mockito.when(model.getLocalRotation()).thenReturn(new Quaternion(0, 0, 0, 0));
         
-        platform = new Platform(new Vector3f(0, 0, 0), Mockito.mock(MainEnvironment.class));
+        platform = new Platform(new Vector3f(0, 0, 0), Mockito.mock(MainEnvironment.class), Mockito.mock(CarrierAssigner.class));
```

Comment:
CheckStyle is complaining this line is too long, please break it up into two lines.